packages/camera/camera/android/src/main/java/io/flutter/plugins/camera/Camera.java

```diff
@@ -183,15 +192,20 @@ private void prepareMediaRecorder(String outputFilePath) throws IOException {
   }
 
   @SuppressLint("MissingPermission")
-  public void open() throws CameraAccessException {
+  public void open(String imageFormatGroup) throws CameraAccessException {
     pictureImageReader =
         ImageReader.newInstance(
             captureSize.getWidth(), captureSize.getHeight(), ImageFormat.JPEG, 2);
 
+    Integer imageFormat = supportedImageFormats.get(imageFormatGroup);
+    if (imageFormat == null) {
+      Log.w(TAG, "The selected imageFormatGroup is not supported by Android. Defaulting to yuv420");
```

Comment:
I'm confused by this `TAG` value. I might be misunderstanding it, but classes typically have a value like this at the top
```
private static final String TAG = 'MyClassName`;
```

But, this class doesn't have that. It looks like your using the `TAG` value from the static import added above:
```
import static android.content.ContentValues.TAG;
```

Is this intentional?

Reply:
No this is not on purpose and is a mistake, will correct it now.